src/test/librbd/fsx.cc

```diff
@@ -1416,9 +1402,9 @@ do_clone()
 	++num_clones;
 
 	if (randomize_striping) {
-		order = 18 + get_random() % 8;
-		stripe_unit = 1ull << (order - 1 - (get_random() % 8));
```

Comment:
Thanks for taking a look. Yeah tossing get_random() wasn't a good idea. I'll put it back.